### PR TITLE
Add rgit to Software Development - Project Management

### PR DESCRIPTION
**Pull request by w4**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Miscellaneous`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another project, please provide/link to a clear list of differences between both.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Comment by KieranRobson** [comment]
> https://github.com/w4/rgit/commit/afaae25fb783d32ca1a2c3a6b39c0604a6437c77
>  Any software project you are adding was first released more than 4 months ago.

Initial Commit July. Feel free to resubmit when this criteria has been met.

I would also recommend adding deployment / installation instructions.

**Comment by KieranRobson** [comment]
> that's July '22 so 13 months ago

Woops! 

I have reopened this PR. Deplotment details would be great and would meet the ` Any software project you are adding has working installation instructions.` criteria for this list.

**Comment by w4** [comment]
Thanks, that's all done now

**Review by KieranRobson — CHANGES_REQUESTED** [review]
> Added installation instructions from source, for NixOS & for Docker

Quick change to be made!

File: README.md [hunk 1]
```diff
@@ -1922,6 +1922,7 @@ _Related: [Ticketing](#ticketing), [Task Management & To-do Lists](#task-managem
 - [Plane](https://plane.so) - Open Source JIRA, Linear and Height Alternative. Plane helps you track your issues, epics, and product roadmaps in the simplest way possible. ([Demo](https://app.plane.so), [Source Code](https://github.com/makeplane/plane)) `Apache-2.0` `Docker`
 - [ProjeQtOr](https://www.projeqtor.org/) - A complete, mature, multi-user project management system with extensive functionality for all phases of a project. ([Demo](https://demo.projeqtor.org/), [Source Code](https://sourceforge.net/p/projectorria/code/HEAD/tree/branches/)) `AGPL-3.0` `PHP`
 - [Redmine](https://www.redmine.org/) - Redmine is a flexible project management web application. ([Source Code](https://svn.redmine.org/redmine/)) `GPL-2.0` `Ruby`
+- [rgit](https://github.com/w4/rgit) - An ultra-fast & lightweight cgit clone. ([Demo](https://git.inept.dev/)) `WTFPL` `Rust`
```
[the remark this answers is not in the document]
Comment (w4) [reply README.md:1925]: Updated

**Review by KieranRobson — APPROVED** [review]
Looks good to me!

**Comment by nodiscc** [comment]
>  Any software project you are adding was first released more than 4 months ago.

A clearly identified release would be nice (git tag or github release), in order to fulfill this requirement. There is a `0.1.0` version identifier in `cargo.toml` that seems to be there since the initial commit, so tagging this commit as `0.1.0` would be fine. In addition, this lets users decide when they actually want to upgrade, instead of running a development/unstable version by default.

Approving anyway.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**